dangerfile.js

```diff
@@ -0,0 +1,181 @@
+/* eslint-disable no-console */
+// inspire by reacts dangerfile
+// danger has to be the first thing required!
+const { danger, markdown } = require('danger');
+const { exec } = require('child_process');
+const { loadComparison } = require('./scripts/sizeSnapshot');
+
+const parsedSizeChangeThreshold = 300;
+const gzipSizeChangeThreshold = 100;
+
+/**
+ * executes a git subcommand
+ * @param {any} args
+ */
+function git(args) {
+  return new Promise((resolve, reject) => {
+    exec(`git ${args}`, (err, stdout) => {
+      if (err) {
+        reject(err);
+      } else {
+        resolve(stdout.trim());
+      }
+    });
+  });
+}
+
+const upstreamRemote = 'danger-upstream';
+
+async function cleanup() {
+  await git(`remote remove ${upstreamRemote}`);
+}
+
+/**
+ * creates a callback for Object.entries(comparison).filter that excludes every
+ * entry that does not exceed the given threshold values for parsed and gzip size
+ *
+ * @param {number} parsedThreshold
+ * @param {number} gzipThreshold
+ */
+function createComparisonFilter(parsedThreshold, gzipThreshold) {
+  return ([, { parsed, gzip }]) => {
```

Comment:
Personally preference I would like to share: I tend to avoid destructuring as much as possible as it often obfuscates the object we are manipulating. In this case, I have no idea what's the second array item.
(I find code edition to be harder)

Reply:
Agreed. Will change

Reply:
@oliviertassinari Is this clearer: 006c7da08a631e610cc1cd599499e56d1fa75826?